docs/quick-start/receivers/axis-thor.md

```diff
@@ -19,39 +20,47 @@ There are Flight Controllers that put their Receiver UART's RX pads Low, which i
 
 <figure markdown>
 ![pull up](../../assets/images/pull-up.png)
+<figcaption>Wiring up receiver</figcaption>
 </figure>
 
 ## Configuring your Flight Controller
 
-See this [page](configuring-fc.md) on how your flight controller should be configured. These settings apply on both INAV and Betaflight (and other flight controller software).
+To configure your flight controller properly, please go through [Configure FC page](configuring-fc.md). These settings apply on INAV, Betaflight and other flight controller software.
 
 Ports Tab should be setup so that Serial RX is on the UART where you have soldered the receiver.
 
-Receiver protocol is `CRSF`, with `serialrx_inverted = off` and `serialrx_halfduplex = off`.
+Receiver protocol is `CRSF` with `serialrx_inverted = off` and `serialrx_halfduplex = off`.
 
 The next step will not be able to proceed properly and you'll have issues later if any of these are set differently. Once you have configured your Flight Controller software, close its Configurator and unplug-replug the USB cable from the FC or your computer. This will refresh the connection and you'll be ensured that the port is not busy (of high importance with the Passthrough Flashing Method).
 
 ## Flashing via WiFi
 
-Target: `AXIS_THOR_2400_RX_via_WIFI`
+- Target: `AXIS_THOR_2400_RX_via_WIFI`
 
-Device Category: `AXIS 2.4GHz`
+- Device Category: `AXIS 2.4GHz`
 
-Device : `AXIS THOR 2400RX`
+- Device: `AXIS THOR 2400RX`
```

Comment:
In my opinion, grouping/listing improves readability, and content gains meaning. That's why I updated all of the receivers, but in case you think it is unnecessary or not needed, I can revert. :)